nucliadb_sdk/nucliadb_sdk/utils.py

```diff
@@ -123,8 +124,10 @@ def get_or_create(
 
 def delete_kb(slug: str, nucliadb_base_url: Optional[str] = "http://localhost:8080"):
     kb = get_kb(slug, nucliadb_base_url)
-    if kb is None or kb.client.url is None:
+    if kb.client.url is None:
         raise AttributeError("URL should not be none")
+    if not kb:
+        raise NotFoundError("KB does not exists.")
```

Comment:
I think the order of the checks should be the opposite. Otherwise if kb is None, we would crash.